src/components/PopUpWithConfirmation.js

```diff
@@ -0,0 +1,42 @@
+import Popup from "./Popup.js";
```

Comment:
* According to the checklist you need to make all files with classes in `UppperCase` https://snipboard.io/YyEkOP.jpg 


If the file name doesn't change on github, then you need to clear the git cache:

```bash

git rm -r --cached .
git add .
git commit -am 'git cache cleared'
git push

```

Reply:
on my end every file starts with the Uppercase
![Screenshot 2024-09-10 104443](https://github.com/user-attachments/assets/55f6bf7c-e373-4ce7-bbba-570c0a4faab5)

Reply:
> on my end every file starts with the Uppercase

It's on your computer, but check your **github repo**.


If the file name doesn't change on github, then you need to clear the git cache:

```bash

git rm -r --cached .
git add .
git commit -am 'git cache cleared'
git push

```

Reply:
I did remove the cache and when I push it, it says everything is up to date, like on the example that I did on the picture before